dev/shim/dune

```diff
@@ -0,0 +1,11 @@
+(rule
+ (targets coqtop)
```

Comment:
This line seems to be in contradiction with the `(with-outputs-to coqtop-prelude` line. Moreover, I don't think you need this line in the presence of the other one.

Reply:
Fixed thanks!